src/main/java/org/pentaho/metadata/query/model/util/QueryXmlHelper.java

```diff
@@ -560,7 +561,9 @@ protected void addOrderByFromXmlNode( Query query, Element orderElement ) throws
     }
 
     if ( ( view_id != null ) && ( column_id != null ) ) {
-      addOrderBy( query, view_id, column_id, aggregation, orderType );
+      try {
+        addOrderBy( query, view_id, column_id, aggregation, orderType );
+      } catch ( BusinessColumnException ignored ) {}
```

Comment:
The logic sounds good, but be aware that it's not a good practice to use exceptions as code logic, and this seems to be the case, since the exception is ignored and not even logged. So I'd like to raise the question: Is it impossible/impractical to make the check here as an if before the call to `addOrderBy`?

Something like:
```suggestion
      LogicalColumn column = category.findLogicalColumn( columnId );
      if ( column == null ) {
        return;
      }

      addOrderBy( query, view_id, column_id, aggregation, orderType );
```

Reply:
Already tested it, works the same. Thanks! I have updated the PR